bin/validate-json

```diff
@@ -198,21 +210,21 @@ try {
     $urlSchema = $resolver->resolve($pathSchema, $urlData);
 
     if (isset($arOptions['--dump-schema-url'])) {
-        echo $urlSchema . "\n";
+        output($urlSchema . "\n");
         exit();
     }
 } catch (Exception $e) {
-    echo "Error loading JSON schema file\n";
-    echo $urlSchema . "\n";
-    echo $e->getMessage() . "\n";
+    output("Error loading JSON schema file\n");
+    output($urlSchema . "\n");
+    output($e->getMessage() . "\n");
     exit(2);
 }
 $refResolver = new JsonSchema\SchemaStorage($retriever, $resolver);
 $schema = $refResolver->resolveRef($urlSchema);
 
 if (isset($arOptions['--dump-schema'])) {
```

Comment:
GitHub highlighted the wrong line - I'm referring to `if (isset($arOptions['--dump-schema']))`